README.md

```diff
@@ -31,4 +31,8 @@ The source files in this repo are licensed to you under the EUPL 1.2. You can do
 ## More information about Valtimo
 For more information check the following links.
 - Website: https://www.valtimo.nl
-- Documentation: https://docs.valtimo.nl
\ No newline at end of file
+- Documentation: https://docs.valtimo.nl
+
+## Getting started
+* Clone `https://github.com/valtimo-platform/valtimo-docker-profiles` and build the container for your desired edition
+* Then build the back-end libraries and run the gradle `bootRun` for your desired edition (for the gzac edition this is `app/gzac/application/bootRun`)
```

Comment:
I believe running bootRun will also build the project. So running it separately is not neccasary.